controllers/setup.go

```diff
@@ -54,7 +54,7 @@ func (r *KymaReconciler) SetupWithManager(mgr ctrl.Manager,
 	controllerBuilder = controllerBuilder.Watches(&source.Kind{Type: &moduleManagerV1alpha1.Manifest{}},
```

Comment:
can we just append this `.Watches` directly after line 52, but not reassign this `controllerBuilder`?

Reply:
dropped the commit